tornado/test/netutil_test.py

```diff
@@ -96,6 +96,28 @@ def tearDown(self):
         super(BlockingResolverErrorTest, self).tearDown()
 
 
+@skipIfNoNetwork
```

Comment:
If it's falling through to the network, then the override isn't working.

Reply:
Removed